x/icacallbacks/keeper/keeper.go

```diff
@@ -105,15 +105,15 @@ func (k Keeper) CallRegisteredICACallback(ctx sdk.Context, modulePacket channelt
 	// fetch the callback function
 	callbackHandler, err := k.GetICACallbackHandler(module)
 	if err != nil {
-		k.Logger(ctx).Info("CallRegisteredICACallback", "err", err)
+		k.Logger(ctx).Error(fmt.Sprintf("CallRegisteredICACallback %s", err.Error()))
 		return err
 	}
 	// call the callback
 	if callbackHandler.HasICACallback(callbackData.CallbackId) {
 		// if acknowledgement is empty, then it is a timeout
 		err := callbackHandler.CallICACallback(ctx, callbackData.CallbackId, modulePacket, acknowledgement, callbackData.CallbackArgs)
 		if err != nil {
-			k.Logger(ctx).Info("CallRegisteredICACallback", "err", err)
+			k.Logger(ctx).Error(fmt.Sprintf("CallRegisteredICACallback %s", err.Error()))
```

Comment:
nit: should we put a more specific error here? Something like "no callback handler found for module %s" and "error executing {callbackId} callback"

Reply:
done